phoenix-core/src/main/java/org/apache/phoenix/schema/PTableImpl.java

```diff
@@ -1124,9 +1129,11 @@ public PColumn getColumnForColumnQualifier(byte[] cf, byte[] cq) throws ColumnNo
         private final long ts;
         private final boolean hasOnDupKey;
         // map from column name to value 
-        private Map<PColumn, byte[]> columnToValueMap; 
+        private Map<PColumn, byte[]> columnToValueMap;
+        // map from the column family name to the list of dynamic columns in that column family
+        private Map<String, List<PColumn>> colFamToDynamicColumnsMapping;
 
-        public PRowImpl(KeyValueBuilder kvBuilder, ImmutableBytesWritable key, long ts, Integer bucketNum, boolean hasOnDupKey) {
+        PRowImpl(KeyValueBuilder kvBuilder, ImmutableBytesWritable key, long ts, Integer bucketNum, boolean hasOnDupKey) {
```

Comment:
Question: Why the change from public to protector?

Reply:
@gokceni This method does not require public scope hence I changed it to package-private